src/encoded/schemas/mixins.json

```diff
@@ -432,7 +432,7 @@
                     "sarah-teichmann|",
                     "sten-linnarsson|",
                     "lattice",
-                    "):[a-zA-Z\\d_$.+!*,()'-]+(?:\\s[a-zA-Z\\d_$.+!*,()'-]+)*$"
+                    "):[a-zA-Zü\\d_$.+!*,()'-]+(?:\\s[a-zA-Zü\\d_$.+!*,()'-]+)*$"
```

Comment:
We need this to capture several new characters. Check out the Users tab on Initial Submission sheet for some...Guigó, Gonçalo, Körner are the ones I noticed on a quick scan. Ideally, we get it ready now for those AND any other non-English characters that might come

Reply:
I've changed the regex to be more permissible for non-English characters. Not sure why the regex pattern is repeated within the parenthesis, but I just left it that way.